### PR TITLE
Completed exercise react-refresher. - Jesus Araujo

### PR DESCRIPTION
**Pull request by j-alejandro-araujo**
Kapture 2023-06-26 at 10 32 06

### CODE REVIEW

File: react-refresher/my-app/src/App.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+      </header>
+      <span className="counter_number">{counter}</span>
+      <div className="buttons-container">
+        <button className="down-btn" onClick={decrease}>
+          DOWN
+        </button>
+        <button className="up-btn" onClick={increase}>
+          UP
+        </button>
+      </div>
```
Comment (shawnkost) [inline react-refresher/my-app/src/App.js:28]: This should be a separate component

File: react-refresher/my-app/src/App.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+function App() {
+  const [counter, setCounter] = useState(0);
+
+  const increase = () => {
+    setCounter((count) => count + 1);
+  };
+
+  const decrease = () => {
+    setCounter((count) => count - 1);
+  };
```
Comment (shawnkost) [inline react-refresher/my-app/src/App.js:13]: No need for a callback function. Just do `setCounter(count + 1)`

**Review by shawnkost — CHANGES_REQUESTED** [review]
Couple changes here Jesus!

**Comment by j-alejandro-araujo** [comment]
Adjustments complete!

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**